packages/salesforcedx-vscode-apex/src/messages/i18n.ts

```diff
@@ -56,5 +56,7 @@ export const messages = {
     'Unable to activate the Apex Language Server',
   sobject_refresh_all: 'All SObjects',
   sobject_refresh_custom: 'Custom SObjects',
-  sobject_refresh_standard: 'Standard SObjects'
+  sobject_refresh_standard: 'Standard SObjects',
+  sobjects_refresh_needed: "You don't have any sObjects cached locally.To take advantage of autocompletion for sObjects in Apex code, run SFDX: Refresh sObject Definitions.",
```

Comment:
```suggestion
  sobjects_refresh_needed: "You don't have any sObjects cached locally. To take advantage of autocompletion for sObjects in Apex code, run SFDX: Refresh SObject Definitions.",
```

Missing space and updating the command reference to use the Capital S spelling of sObject because the command in the command palette uses that version. The reason for this I believe is because when referring to an sObject Apex type, the class it inherits from is called `SObject`, which is a built-in type and follows the Apex type convention of pascalcase. When talking about sObjects more generally in the context of the whole platform, the camelcase version is used. @ekapner does that sound right?

Reply:
That does sound correct and I could go either way.